Net/src/Socket.cpp

```diff
@@ -167,7 +168,7 @@ int Socket::select(SocketList& readList, SocketList& writeList, SocketList& exce
 	}
 
 	struct epoll_event eventsOut[epollSize];
-	memset(eventsOut, 0, sizeof(eventsOut));
+	memset(eventsOut, 0, sizeof(epoll_event) * epollSize );	
```

Comment:
gcc with -std=c++11 sizeof(eventsOut) gives bytes number
gcc with -std=c++14 sizeof(eventsOut) gives elements number
changed memset to keep it safe